### DISCUSSION TITLE
Nach Update keine Host's mehr vorhanden

### DISCUSSION THREAD

**Discussion by DrGalvany** [NginxProxyManager/nginx-proxy-manager#2874] [General]
Hallo, habe auf v2.10.2 aktualisiert.
Es sind in der UI keine Hosts mehr ersichtlich, also alles leer wie ein frisches System.
In der Oberfläche konnte ich mich auch nur mit den Standarddaten einloggen.
Funktionieren tun anscheinend noch alle.
Ich nutze docker-compose.yml.
Wenn noch Daten fehlen, reiche ich die gerne nach, bin da leider kein Profi.

Danke
Micha

**Comment by ZIOOIZ8** [comment]
Hello, I've the same problem, too. I tried to solve it, but all ways faild. Unfortunately no one made a easy turtorial, how can it be fixed and I'm not a database specialist.

**Comment by Vandroiy78** [comment]
I had the same issue at first. Then I noted that the config file from previous versions seems to be no longer in use. The connection parameters for the database must now be provided as environment parameters in the docker container.

For example (taken from the online documentation):
      DB_MYSQL_HOST: "db"
      DB_MYSQL_PORT: 3306
      DB_MYSQL_USER: "npm"
      DB_MYSQL_PASSWORD: "npm"
      DB_MYSQL_NAME: "npm"

https://nginxproxymanager.com/setup/#running-the-app

Hope this helps.

Vandroiy.
